### ISSUE TITLE
Error upon fresh start

### ISSUE THREAD

**Issue by kerbe** [apiel/test-crawler#17]
Using npm version 1.0.0-alpha.0. Startup works fine, but once I open browser, it spits two errors, one for path argument type and another for not finding projects apparently.

Log of those:
```
Start test crawler

> test-crawler@1.0.0-alpha.0 isomor:server /usr/local/share/.config/yarn/global/node_modules/test-crawler
> node ../isomor-server/dist/bin/server.js

• info Starting server.
• info Created endpoints: [
  '/isomor/test-crawler/server-service/getSettings',
  '/isomor/test-crawler/server-service/getCrawlers',
  '/isomor/test-crawler/server-service/loadProject',
  '/isomor/test-crawler/server-service/loadProjects',
  '/isomor/test-crawler/server-service/saveProject',
  '/isomor/test-crawler/server-service/getCrawler',
  '/isomor/test-crawler/server-service/getPages',
  '/isomor/test-crawler/server-service/getPins',
  '/isomor/test-crawler/server-service/getPin',
  '/isomor/test-crawler/server-service/setCode',
  '/isomor/test-crawler/server-service/getCode',
  '/isomor/test-crawler/server-service/getCodes',
  '/isomor/test-crawler/server-service/getThumbnail',
  '/isomor/test-crawler/server-service/pin',
  '/isomor/test-crawler/server-service/setZoneStatus',
  '/isomor/test-crawler/server-service/setZonesStatus',
  '/isomor/test-crawler/server-service/setStatus',
  '/isomor/test-crawler/server-service/startCrawlerFromProject',
  '/isomor/test-crawler/server-service/startCrawlers'
]
• info Add static folder /usr/local/share/.config/yarn/global/node_modules/test-crawler/../test-crawler/build
• success Server listening on port 3005!
• info Find API documentation at http://127.0.0.1:3005/api-docs
• GET / 304 5.753 ms - -
• GET /static/css/main.1b7acf8b.chunk.css 304 3.890 ms - -
• GET /static/css/2.ff8bd605.chunk.css 304 4.144 ms - -
• GET /static/js/2.54874c91.chunk.js 304 3.331 ms - -
• GET /static/js/main.10031846.chunk.js 304 0.901 ms - -
• ERR TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string. Received type object
    at validateString (internal/validators.js:112:11)
    at Object.join (path.js:1039:7)
    at CrawlerProvider.<anonymous> (/usr/local/share/.config/yarn/global/node_modules/test-crawler/dist-server/server/lib/index.js:179:42)
    at Generator.next (<anonymous>)
    at /usr/local/share/.config/yarn/global/node_modules/test-crawler/dist-server/server/lib/index.js:8:71
    at new Promise (<anonymous>)
    at __awaiter (/usr/local/share/.config/yarn/global/node_modules/test-crawler/dist-server/server/lib/index.js:4:12)
    at CrawlerProvider.getAllCrawlers (/usr/local/share/.config/yarn/global/node_modules/test-crawler/dist-server/server/lib/index.js:178:16)
    at Object.getCrawlers (/usr/local/share/.config/yarn/global/node_modules/test-crawler/dist-server/server/service.js:21:28)
    at Object.<anonymous> (/usr/local/share/.config/yarn/global/node_modules/isomor-server/dist/lib/entrypoint.js:49:61) {
  code: 'ERR_INVALID_ARG_TYPE'
}
• GET /isomor/test-crawler/server-service/getCrawlers 500 5.700 ms - 64
• ERR [Error: ENOENT: no such file or directory, open '/home/pptruser/app/base/project.json'] {
  errno: -2,
  code: 'ENOENT',
  syscall: 'open',
  path: '/home/pptruser/app/base/project.json'
}
• GET /isomor/test-crawler/server-service/loadProjects 500 12.623 ms - 78
• GET /manifest.json 304 0.581 ms - -
```

**Comment by kerbe** [comment]
As +New button is visible for creating project, if I try to add project I get code 500 error in popup, and in console it states this:
```
• ERR TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string. Received type object
    at validateString (internal/validators.js:112:11)
    at Object.join (path.js:1039:7)
    at CrawlerProvider.<anonymous> (/usr/local/share/.config/yarn/global/node_modules/test-crawler/dist-server/server/lib/index.js:201:33)
    at Generator.next (<anonymous>)
    at /usr/local/share/.config/yarn/global/node_modules/test-crawler/dist-server/server/lib/index.js:8:71
    at new Promise (<anonymous>)
    at __awaiter (/usr/local/share/.config/yarn/global/node_modules/test-crawler/dist-server/server/lib/index.js:4:12)
    at CrawlerProvider.saveProject (/usr/local/share/.config/yarn/global/node_modules/test-crawler/dist-server/server/lib/index.js:197:16)
    at Object.saveProject (/usr/local/share/.config/yarn/global/node_modules/test-crawler/dist-server/server/service.js:33:28)
    at Object.<anonymous> (/usr/local/share/.config/yarn/global/node_modules/isomor-server/dist/lib/entrypoint.js:49:61) {
  code: 'ERR_INVALID_ARG_TYPE'
}
• POST /isomor/test-crawler/server-service/saveProject 500 1.589 ms - 64
```

**Comment by kerbe** [comment]
Other problems seems to be fixed in v1.0.1, but there remains error about missing projects:
```
• success Server listening on port 3005!
• info Find API documentation at http://127.0.0.1:3005/api-docs
• GET / 304 4.899 ms - -
• GET /static/css/2.0b61d4ae.chunk.css 304 1.524 ms - -
• GET /static/css/main.56451939.chunk.css 304 1.499 ms - -
• GET /static/js/2.002e7bbf.chunk.js 304 1.575 ms - -
• GET /static/js/main.5f9bb723.chunk.js 304 2.180 ms - -
• GET /manifest.json 304 0.680 ms - -
• WS req /isomor/test-crawler/server-service/loadProjects
• WS 500 /isomor/test-crawler/server-service/loadProjects [Error: ENOENT: no such file or directory, open '/home/pptruser/app/preset/project.json'] {
  errno: -2,
  code: 'ENOENT',
  syscall: 'open',
  path: '/home/pptruser/app/preset/project.json'
}
```

Once I create project, it saves it, but doesn't apparently save/link project.json in right places. It created new hash folder in root of `PAGES_FOLDER`, which contains project.json. Then it complained that it doesn't find project.json from `base`, `crawl` and `preset` folders, in that order. I copied that project.json in each of those, and then I got new error (so first copied to base, got error that it is missing from crawl, copied there, then error that it is missing from preset). After that no errors, but project was listed 4 times in web interface, all leading to same place. There was some error while trying to run project on first time (didn't catch that), and different problem when I tried running it again:
```
• WS 500 /isomor/test-crawler/server-service/getCrawler SyntaxError: /home/pptruser/app/9a132c9d50884fdcb90bb3ee278c0770/crawl/1580799156/_.json: Unexpected end of JSON input
    at JSON.parse (<anonymous>)
    at /usr/local/share/.config/yarn/global/node_modules/jsonfile/index.js:33:18
    at /usr/local/share/.config/yarn/global/node_modules/graceful-fs/graceful-fs.js:115:16
    at FSReqCallback.readFileAfterClose [as oncomplete] (internal/fs/read_file_context.js:61:3)
```

**Comment by apiel** [comment]
I think most of your problem as been solve in the version v1.1.0. If you keep having problem, feel free to open a new issue.

Thx for you help
